xformers/components/attention/csrc/cuda/mem_eff_attention/iterators/predicated_tile_access_iterator_residual_last.h

```diff
@@ -274,57 +282,18 @@ class PredicatedTileAccessIterator<
   /// tiles
   CUTLASS_DEVICE
   void add_tile_offset(TensorCoord const& tile_offset) {
-    if (is_residue_tile_) {
-      the_predicates.thread_offset_ += the_predicates.residue_offset_;
-
-      the_predicates.compute_predicates_(the_predicates.extent_, true);
-
-      Layout layout(params_.stride_);
-
-      if (!Gather) {
-        add_pointer_offset(layout(the_predicates.residue_offset_));
-
-        if (kAdvanceRank) {
-          pointer_ +=
-              params_.inc_advance_ * LongIndex(tile_offset.strided() - 1);
-          pointer_ += Shape::kContiguous * tile_offset.contiguous();
-        } else {
-          pointer_ +=
-              params_.inc_advance_ * LongIndex(tile_offset.contiguous() - 1);
-          pointer_ += Shape::kStrided * tile_offset.strided();
-        }
+    if (!Gather) {
```

Comment:
Same comment here, this is the only thing that we changed (in addition to the `set_residual_tile`) but we need to copy the whole file. :-/